server/ast/expr.go

```diff
@@ -571,3 +578,35 @@ func nodeExpr(node tree.Expr) (vitess.Expr, error) {
 		return nil, fmt.Errorf("unknown expression: `%T`", node)
 	}
 }
+
+func translateConvertType(convertType *vitess.ConvertType) (*vitess.ConvertType, error) {
```

Comment:
No issue with adding the `translateConvertType` here since it's going from `vitess.ConvertType` to itself, which is a new type of transformation. We should make a new entry in the [contribution guide](https://github.com/dolthub/doltgresql/blob/main/CONTRIBUTING.md#serverast) though, probably right at `3` describing what a `translate` function is, since `1` and `2` describe the other types. Also need to update `2` since it mentions adding other transforms. Also, should add a function comment.

Reply:
I'll do this as a followup